<launch>

	<node name="joy_node" pkg="joy" type="joy_node"/>

	<node name="teleop_drive" pkg="illini_racecar" type="teleop_drive.py"/>

	<node name="drive_controller_node" pkg="illini_racecar" type="drive_controller_node"/>

	<node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen">
		<param name="port" value="/dev/ttyACM0"/>	
		<param name="baud" value="57600"/>	
	</node>
	
</launch>
